webui/react/src/pages/TrialDetails/TrialInfoBox.tsx

```diff
@@ -2,26 +2,49 @@ import Card from 'hew/Card';
 import { useModal } from 'hew/Modal';
 import { Loadable, Loaded, NotLoaded } from 'hew/utils/loadable';
 import { isEqual } from 'lodash';
+import { useObservable } from 'micro-observables';
 import React, { useCallback, useEffect, useMemo, useState } from 'react';
 
 import CheckpointModalComponent from 'components/CheckpointModal';
 import ModelCreateModal from 'components/ModelCreateModal';
 import OverviewStats from 'components/OverviewStats';
 import RegisterCheckpointModal from 'components/RegisterCheckpointModal';
+import ResourceAllocationModalComponent from 'components/ResourceAllocationModal';
 import Section from 'components/Section';
 import TimeAgo from 'components/TimeAgo';
-import { getModels } from 'services/api';
+import { SimplifiedNode } from 'pages/ResourcePool/NodeElement';
+import { getModels, getTaskAllocation } from 'services/api';
 import { V1GetModelsRequestSortBy } from 'services/api-ts-sdk';
-import { CheckpointWorkloadExtended, ExperimentBase, ModelItem, TrialDetails } from 'types';
+import clusterStore from 'stores/cluster';
+import {
+  AllocationData,
```

Comment:
`AllocationData` API is call every few seconds. Do we need to call it frequently?